lib/db/schema/patch-091-092.sql

```diff
@@ -0,0 +1,66 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('91');
+
+-- Removes the LOWER(inUid) requirement on the WHERE clause
+CREATE PROCEDURE `account_6` (
+    IN `inUid` BINARY(16)
+)
+BEGIN
+    SELECT
+        a.uid,
+        a.email,
+        a.normalizedEmail,
+        a.emailVerified,
+        a.emailCode,
+        a.kA,
+        a.wrapWrapKb,
+        a.verifierVersion,
+        a.authSalt,
+        a.verifierSetAt,
+        a.createdAt,
+        a.locale,
+        a.lockedAt,
+        COALESCE(a.verifierSetAt, a.createdAt) AS profileChangedAt
```

Comment:
Isn't this the problematic `COALESCE` mentioned in the comment above? We don't want to reintroduce it do we?

Reply:
I believe the problem was with `accountRecord_*` procedure. That has been updated below so that it hopefully doesn't cause the unexpected behavior.

Reply:
Got it, wrong `COALESCE`. And so the `CHARACTER SET utf8 COLLATE utf8_bin` is the fix for that, hence the comment. Makes sense now, sorry!